pkg/logger/logger.go

```diff
@@ -171,9 +170,11 @@ func Start(ctx context.Context, logger *zap.Logger) {
 	if err != nil {
 		log.Fatalf("Error starting pod watcher: %v", err)
 	}
-	informerFactory := k8sInformers.NewSharedInformerFactory(kubernetesClient, time.Minute*30)
-	podInformer := informerFactory.Core().V1().Pods().Informer()
-	podInformer.AddEventHandler(podInformerHandlers(logger))
-	podInformer.Run(ctx.Done())
+
+	for _, podInformer := range utils.GetK8sInformersForNamespaces(kubernetesClient, time.Minute*30, fv1.Pods) {
+		podInformer.AddEventHandler(podInformerHandlers(logger))
+		podInformer.Run(ctx.Done())
```

Comment:
This should be in go routines I guess.

Please use the wait group to spawn informers here.


You can use https://pkg.go.dev/k8s.io/apimachinery@v0.25.4/pkg/util/wait#Group.StartWithChannel

```
var wg wait.Group
for _, podInformer := range informers {
		podInformer.AddEventHandler(podInformerHandlers(logger))
                wg.StartWithChannel(ctx.Done(), podInformer.Run)
}
wg.Wait()
logger.Error("Stop watching pod changes")


```